sdk/src/main/java/io/radar/sdk/RadarActivityLifecycleCallbacks.kt

```diff
@@ -20,6 +20,7 @@ internal class RadarActivityLifecycleCallbacks(
     private val fraud: Boolean = false
 ) : Application.ActivityLifecycleCallbacks {
     private var count = 0
+    private var isFirstCreate = true
```

Comment:
I'm still a little confused on why count alone is not sufficient and we need the extra `isFirstCreate` flag.